src/org/sosy_lab/java_smt/solvers/boolector/BoolectorAbstractProver.java

```diff
@@ -82,6 +84,12 @@ public void close() {
    */
   @Override
   public boolean isUnsat() throws SolverException, InterruptedException {
+    try {
+      Generator.dumpSMTLIB2();
```

Comment:
This will write the generated SMTLIB2 file to a file called `out.smt2`. We may want add a method to get the script as a string instead.